docs/components/Balance.md

```diff
@@ -26,3 +32,26 @@ import { Balance } from "~~/components/scaffold-stark";
 | ------------------------ | -------- | ------------- | ------------------------------------------------------------------------------------------------------------------------- |
 | **address**              | `string` | `undefined`   | Address in `0x___` format, it will resolve its Starknet Domain if it has one associated(coming soon).                     |
 | **className** (optional) | `string` | `""`          | Prop to pass additional CSS styling to the component. You can use Tailwind / daisyUI classes like `text-3xl` for styling. |
+| **usdMode** (optional) | `boolean` | `false`          | If true, the balance is displayed in USD. Otherwise, it defaults to ETH/STRK. |
+
+
+## Example
+
+The `Balance` component can be easily customized using Tailwind or daisyUI classes. Here’s a more detailed example of its usage:
+
+```tsx
+<Balance
+  address="0x34aA3F359A9D614239015126635CE7732c18fDF3"
+  className="text-lg text-gray-700 mt-4"
+  usdMode={true}
+/>
+```
+
+This example showcases how to apply additional styling to the component and enables the USD display mode.
+
+## Notes
```

Comment:
Appreciate the screenshot! I think we can remove the "Notes" header and it will look sweet (since it is redundant with the info text), what do you think?